doc/tutorial/extending_theano.txt

```diff
@@ -259,7 +259,7 @@ Op Example
 
         def make_node(self, x):
             # check that the theano version has support for __props__
-            assert hasattr(self, '_props')
+            assert hasattr(self, '__props__')
```

Comment:
Well, the original reason why "_props" was there is not explained at the tutorial and it's a bit confusing. If it's fine to get rid of this, my opinion would be to do so.

I'm not the one making the judgment call about whether it's time or not to get rid of this. Which one do you want ?